src/server/game/Entities/Creature/TemporarySummon.cpp

```diff
@@ -238,6 +238,9 @@ void TempSummon::InitStats(uint32 duration)
 
     if (m_Properties->GetFlags().HasFlag(SummonPropertiesFlags::SummonFromBattlePetJournal))
         RemoveNpcFlag(UNIT_NPC_FLAG_WILD_BATTLE_PET);
+
+    if (sObjectMgr->GetCreatureSummonedData(GetEntry()))
```

Comment:
remove this, handle with creature_template_addon

Reply:
via auraID?

Reply:
done.